R/geom-tile.r

```diff
@@ -14,8 +14,6 @@
 #' @inheritParams geom_point
 #' @export
 #' @examples
-#' # The most common use for rectangles is to draw a surface. You always want
-#' # to use geom_tile here because it's so much faster, and produces
 #' # smaller output when saving to PDF
```

Comment:
please delete as well

Reply:
more generally, please look at "git diff" or "Files Changed" tab to review the changes that you propose